src/Command/ValidateCommand.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Overblog\GraphQLBundle\Command;
+
+use GraphQL\Error\InvariantViolation;
+use GraphQL\Error\Warning;
+use Overblog\GraphQLBundle\Resolver\TypeResolver;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+final class ValidateCommand extends Command
+{
+    use RequestExecutorLazyLoaderTrait;
+
+    private $typeResolver;
+
+    public function __construct(TypeResolver $typeResolver)
+    {
+        parent::__construct();
+        $this->typeResolver = $typeResolver;
```

Comment:
It seems you don't used this anymore, I'm a right? Can you remove it and construct if this is the case.

Reply:
Right, I removed it.